src/sentry/sentry_apps/api/endpoints/sentry_app_publish_request.py

```diff
@@ -65,7 +72,8 @@ def post(self, request: Request, sentry_app) -> Response:
         org_slug = "<unknown>" if org_mapping is None else org_mapping.slug
         message = f"User {request.user.email} of organization {org_slug} wants to publish {sentry_app.slug}\n"
 
-        for question_pair in request.data.get("questionnaire"):
+        questionnaire: Iterable[dict[str, str]] = request.data.get("questionnaire", [])
+        for question_pair in questionnaire:
             message += "\n\n>{}\n{}".format(question_pair["question"], question_pair["answer"])
```

Comment:
Should we have a guard here for `question` and `answer` keys being defined? Right now it look like we could hit a KeyError here with bad input.